app/presenters/publishing_api/editionable_worldwide_organisation_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module PublishingApi
+  class EditionableWorldwideOrganisationPresenter
+    include Rails.application.routes.url_helpers
+    include ActionView::Helpers::UrlHelper
+
+    attr_accessor :item, :update_type, :state
+
+    def initialize(item, update_type: nil, state: "published")
+      self.item = item
+      self.update_type = update_type || "major"
+      self.state = state
+    end
+
+    delegate :content_id, to: :item
+
+    def content
+      content = BaseItemPresenter.new(
+        item,
+        update_type:,
+      ).base_attributes
+
+      content.merge!(
+        details: {
+          logo: {
+            crest: "single-identity",
+          },
+        },
+        document_type: item.class.name.underscore,
+        public_updated_at: item.updated_at,
+        rendering_app: Whitehall::RenderingApp::GOVERNMENT_FRONTEND,
+        schema_name: "worldwide_organisation",
+      )
+      content.merge!(PayloadBuilder::PolymorphicPath.for(item))
+    end
```

Comment:
Do we need to add in the analytics identifier (like [this](https://github.com/alphagov/whitehall/blob/b814a4ebd2600285ddd226ca5a135e156638199a/app/presenters/publishing_api/worldwide_organisation_presenter.rb#L42C11-L42C11) in the old presenter)?

If we can't add it yet, can you make sure there's a card written so we remember to add it in.

Reply:
Analytics identifier was defined in the old model, so I think it falls under the scope of the work you are doing, or at least subsequent work.

Reply:
Cool.  Could you write a Trello card to add the analytics identifier, as it won't be covered under the associations card I'm currently working on.